README.md

```diff
@@ -112,16 +113,54 @@ Write a conditional statement that...
 8.  Write your javascript solution below
 ```js
 // your answer here
+function factors(num)  {
+    let output = '';
+    output += num % 3 === 0 ? 'Fizz' : '';
+    output += num % 5 === 0 ? 'Buzz' : '';
+    return output ? output : num;
+}
+// Trying out the function 
+
+for (let i = 1; i <= 100; i++) {
+   console.log(factors(i))
+}
 ```
-
 #### BONUS
 9.  Research a [loop](https://javascript.info/while-for) so that your condition runs on every number from 0 to 100
 ```js
-// your answer here
+//for
+for (let i = 1; i <= 100; i++) {
+   console.log(i)
+}
+//while
+let i = 1
+while(i <= 100){
+    i++;
+    console.log(i)
+}
+//do while
+let i = 1
+do {
+  i++;
+    console.log(i)
+} while (i <= 100);
+
+
 ```
 10.  Research a [function](https://javascript.info/function-basics) so that your condition runs on every number from 0 to whatever number is passed into the function
 ```js
 // your answer here
+function factors(num)  {
+    let output = '';
+    output += num % 3 === 0 ? 'Fizz' : '';
+    output += num % 5 === 0 ? 'Buzz' : '';
+    return output ? output : num;
+}
+// Trying out the function 
+
+for (let i = 1; i <= 100; i++) {
```

Comment:
the loop should be inside the function, and where the loop ends should be a user input